src/server/scripts/World/mob_generic_creature.cpp

```diff
@@ -27,7 +27,10 @@ EndScriptData */
 #include "ScriptedCreature.h"
 #include "PassiveAI.h"
 
-#define GENERIC_CREATURE_COOLDOWN   5000
+enum GenericCreature
+{
+    GENERIC_CREATURE_COOLDOWN = 5000
```

Comment:
or just dont bother and leave it in enum like it was, less chance to screw up something